lib/hammer_cli_foreman/audit.rb

```diff
@@ -0,0 +1,76 @@
+module HammerCLIForeman
+
+  class Audit < HammerCLIForeman::Command
+
+    resource :audits
+
+    def request_params
+        params = super
+        params['search'] ||= 'time >= "1 month ago"'
+    end
+
+    module PerPageAuditLimit
+
+      def self.included(base)
+        base.extend_help do |h|
+          h.section(_('Override audit record pagination')) do
+            h.text(_('Audit records for past one month are returned by default. This can be paginated further by using --per-page and --page options.'))
+          end
+        end
+      end
+
+    end
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      include PerPageAuditLimit
```

Comment:
If this is the only place where we're going to use the module, I'd vote for deleting the module and just moving the logic to this class.

Reply:
makes sense.